src/controllers/controller.doughnut.js

```diff
@@ -151,6 +151,7 @@ module.exports = DatasetController.extend({
 		var arcs = meta.data;
 		var cutoutPercentage = opts.cutoutPercentage;
 		var circumference = opts.circumference;
+		var chartWeight = chart.data.datasets[me.index].weight ? chart.data.datasets[me.index].weight : 1;
```

Comment:
You could simplify this to:

    var chartWeight = chart.data.datasets[me.index].weight || 1

Reply:
Or maybe `me._getRingWeight(me.index)`